File: Backend/TopSys.TopConWeb.Domain/Entities/Oportunidades/OportunidadeContato.cs
namespace TopSys.TopConWeb.Domain.Entities.Oportunidades
{
    public class OportunidadeContato
    {
        public int Usina { get; set; }
        public int NumeroOportunidade { get; set; }
        public int AnoOportunidade { get; set; }
        public int Sequencia { get; set; }
        public virtual CadastroGeral Funcao { get; set; }
        public int? FuncaoCodigo { get; set; } = 0;
        public int DddTelefone { get; set; }
        public int Telefone { get; set; }
        public int DddCelular { get; set; }
        public int Celular { get; set; }
        public string Email { get; set; }
        public string Nome { get; set; }
    }
}

File: Frontend/TopConWeb-AngularV8/src/app/classes/oportunidade/oportunidade-contato.ts
import { CadastroGeral } from "../cadastro-geral/cadastro-geral";
import { Usina } from "../usina/usina";

export class OportunidadeContato {
    usina: Usina;
    numeroOportunidade: number = 0;
    anoOportunidade: number = 0;
    sequencia: number = 0;
    nome: string = '';

    funcao: CadastroGeral;
    funcaoCodigo: number = 0;

    dddTelefone: number = 0;
    telefone: number = 0;
    dddCelular: number = 0;
    celular: number = 0;

    email: string ='';
}